### DISCUSSION TITLE
Autostart nodejs-poolController on Docker QNAP

### DISCUSSION THREAD

**Discussion by Gouns68** [tagyoureit/nodejs-poolController#838] [App and O/S Config Questions]
Good morning,
I am using NodeJS-pool Controller and the Dashpanel in 2 Docker on my QNAP NAS. When my NAS restarts, I have to connect with the terminal of each Docker to execute the "npm start" command. Does anyone know a solution so that each time a Docker is restarted, this command runs automatically?
Thank you all for your help.

**Comment by tagyoureit** [comment]
Did you see the wiki?
https://github.com/tagyoureit/nodejs-poolcontroller/wiki/Docker

**Comment by Gouns68** [comment]
Thanks for your help. I had seen that, but I hadn't understood that using this method of installation allowed an automatic start (I'm slowly discovering how Dockers work). On the other hand with this method, I don't understand how to customize my config, because in my config.son I don't use USB, but netConnect with a nethost and a netport. Where can I find the detail of the parameters to give in docker-compose.yml to use the parameters that I placed in the config.json?
